docs/usage/cli_changelog.md

```diff
@@ -14,3 +15,23 @@ positional arguments:
 optional arguments:
   -h, --help       show this help message and exit
 ```
+
+## Requirements
+
+The `CHANGELOG.md` file must follow the convention [Keep A Changelog](https://keepachangelog.com/). For example, see this [repository changelog](https://github.com/opengisch/qgis-plugin-ci/blob/master/CHANGELOG.md).  
+If your format is different, you must use a different `changelog_regexp` expression to parse it in your settings.
```

Comment:
This is not possible anymore, no ?

Reply:
I did not take the responsibility to disable this ability, only opened the discussion.

If we agree on disable this, we should raise a DeprecationWarning maybe.

Reply:
Well, as the number of the "extraction" from the regexp (3 before, versus a lot now), existing regexp will break anyway, no ?

So @3nids will need to upgrade the major version number I think.

Reply:
> Well, as the number of the "extraction" from the regexp (3 before, versus a lot now), existing regexp will break anyway, no ?

You're right